pkg/controller/replicaset/replica_set.go

```diff
@@ -681,14 +725,57 @@ func slowStartBatch(count int, initialBatchSize int, fn func() error) (int, erro
 	return successes, nil
 }
 
-func getPodsToDelete(filteredPods []*v1.Pod, diff int) []*v1.Pod {
+// getIndirectlyRelatedPods returns all pods that are owned by any ReplicaSet
+// that is owned by the given ReplicaSet's owner.
+func (rsc *ReplicaSetController) getIndirectlyRelatedPods(rs *apps.ReplicaSet) ([]*v1.Pod, error) {
+	var relatedPods []*v1.Pod
+	seen := map[types.UID]*apps.ReplicaSet{}
```

Comment:
style: use `make(map[types.UID]*apps.ReplicaSet)` as it is the go and Kube convention

Reply:
Done.